tdvt/tdvt/exprtests/standard/expected.setup.string.endswith.null.txt

```diff
@@ -0,0 +1,30 @@
+<results>
+  <test name='ENDSWITH(str2, Null)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.postgres].[TEMP(Test)(2710024337)(0)]</column>
+      </schema>
+      <tuple>
+        <value>%null%</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='ENDSWITH(str1,&apos;&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.postgres].[TEMP(Test)(2210614618)(0)]</column>
+      </schema>
+      <tuple>
+        <value>true</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='ENDSWITH(Num0, Null)'>
+    <error>
+      Reference to undefined field Num0.&#10;Perhaps you meant [num0]?&#10;&#10;    </error>
```

Comment:
We don't want to check in an error as an expected. If a ds doesn't support this test we can add the test case to the excludes list in the ds's .ini file.